src/component/mxgraph/shape/IconPainter.ts

```diff
@@ -502,4 +502,32 @@ export default class IconPainter {
     canvas.close();
     canvas.fillAndStroke();
   }
+
+  // this implementation is adapted from the 'plus square' symbol: https://icons.getbootstrap.com/icons/plus-square/
```

Comment:
:question: @csouchet I don't think we need this as we have a pure custom implementation here. If you want to keep this comment, we should update the BPMN support doc as well
```suggestion
```

Reply:
Yes, I forgot to remove it 😉